src/main/java/com/ning/http/util/AuthenticatorUtils.java

```diff
@@ -215,4 +215,16 @@ private static StringBuilder append(StringBuilder builder, String name, String v
 
         return builder.append(", ");
     }
+
+    public static String getHttpHeaderForAuthScheme(List<String> authenticationHeaders, String authScheme) {
+        if (authenticationHeaders.size() == 1 || authScheme == "NONE") {
+            return authenticationHeaders.get(0);
+        }
+        for (String authenticateHeader : authenticationHeaders) {
+            if (authenticateHeader.length() >= authScheme.length() && authenticateHeader.substring(0,authScheme.length()).equalsIgnoreCase(authScheme)){
```

Comment:
follow the grizzly format: spaces before braces and method parameters

Reply:
also, I think `authenticateHeader.substring(0,authScheme.length())` is the same as `authenticateHeader.substring(authScheme.length())`. consider extracting `authScheme.length()` to a variable since it won't change and you'll be using it on each iteration twice